Core/Core/Network/AuthEndpoint.swift

```diff
@@ -71,6 +77,14 @@ enum AuthEndpoint: EndPointType {
                 "asymmetric_jwt": true
             ]
             return .requestParameters(parameters: params, encoding: URLEncoding.httpBody)
+        case let .socialLogin(externalToken, _, clientId):
+            let params: [String: Encodable] = [
+                "client_id": clientId,
+                "token_type": "jwt",
+                "access_token": externalToken,
+                "asymmetric_jwt": true
```

Comment:
now that we have a `token_type` coming from config, i think we should that value.